config/activitylog.php

```diff
@@ -42,4 +42,10 @@
      * used by the Activity model shipped with this package.
      */
     'table_name' => 'activity_log',
+
+    /*
+     * This is the database connection used to migrate the table and
```

Comment:
You mention the migration but the config value isn't used in the migration?

Reply:
In the first time, I've specified the connection in the migration file:
```
public function up () {  
  Schema::connection(...)->create(..., function (Blueprint $table) {
    ...
}
```
After tests and refactor, I've found its not necessary. So I've remove that but forget to update the comment. Sorry for that!

Reply:
Why shouldn't it be needed? The schema will be created on the default connection if not specified. So it's right that this wouldn't change anything if you migrate before changing the config value. But it gives a hint that it could be that you have to configure something in the migration. We also use the configured table there.

Reply:
I've started a project (1 year half ago) with multiple databases and the first issue I've figured out to solve was the migrations. 
After playing different tests, the only solution I've found was to specifying connection during migration command otherwise nothing worked as expected.
All the migration files are located in a separate folder (one folder per connection) and specified in the migration command `--path={database/migrations/connection_1}`.
```
  $ php artisan migrate --database={db_connection} --path={path_to_migration_files}

  // Example connection name = "conn1"

  $ php artisan migrate --database=conn1 --path="database/migrations/conn1"
 
  // for more info
  $ php artisan migrate --help
```
The version used was L5.6 maybe the current version work differently (I need to re-test on L5.8).

Btw, I got your point and make sense to add it to the migration.

Reply:
If this would be the case it should get reported to the laravel core. I've never used multiple connections in a single app so I can't say anything about this and if there is a bug.
But for us as a package we rely on the working core and try to use as much as possible (where meaningful).

Thanks for adding this back in. I will approve it and check when we can/will release it. Should be within the next week.

Reply:
> If this would be the case it should get reported to the laravel core. 

I don't know if it's a bug! Before to report it to the laravel core, I prefer getting deeper in the codebase and test it again!

> Should be within the next week.

That sounds perfect! 👍